.travis.yml

```diff
@@ -23,4 +22,5 @@ after_success:
 branches:
   only:
     - master
+    - next
```

Comment:
Are we planning to release under next?

Reply:
No. This was added so that CI would run for PRs to the `next` branch. Moreover, there is no harm in keeping the branch in the whitelist and it could be useful in the future.